src/config.ts

```diff
@@ -239,7 +239,7 @@ export const MAX_DATA_ITEM_QUEUE_SIZE = +env.varOrDefault(
   '100000',
 );
 
-// The maximum number of bundles to queue for unbundling before skipping
+// The maximum number of data imports to queue for unbundling/importing before skipping
 export const BUNDLE_DATA_IMPORTER_QUEUE_SIZE = +env.varOrDefault(
```

Comment:
Let's add another configuration option called `VERIFICATION_DATA_IMPORTER_QUEUE_SIZE` and use that for the verification download queue.

Reply:
Don't worry about the docs. I'll go through and clean that up before release 24 (going to restructure `env.md` a bit too).